account_payment_term_extension/tests/test_account_payment_term_multi_day.py

```diff
@@ -0,0 +1,348 @@
+import odoo.tests.common as common
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tests import Form
+
+
+class TestAccountPaymentTermMultiDay(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(
+            context=dict(
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
+        )
+        cls.payment_term_model = cls.env["account.payment.term"]
+        cls.invoice_model = cls.env["account.move"]
+        cls.partner = cls.env["res.partner"].create({"name": "Test Partner"})
+        cls.product = cls.env["product.product"].create({"name": "Test product"})
+        cls.payment_term_0_day_5 = cls.payment_term_model.create(
+            {
+                "name": "Normal payment in day 5",
+                "active": True,
+                "line_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "value": "percent",
+                            "value_amount": 100.0,
+                            "nb_days": 5,
+                            "payment_days": "5",
+                        },
+                    )
```

Comment:
Could you use the [Command](https://github.com/odoo/odoo/blob/370de95190cf07f17281f0ed6365b6806663cac2/odoo/fields.py#L4227-L4256) class? It is more readable.

```suggestion
                   Command.create(
                        {
                            "value": "percent",
                            "value_amount": 100.0,
                            "nb_days": 5,
                            "payment_days": "5",
                        },
                    )
```

Reply:
Done. For all occurrences.